addscore.php

```diff
@@ -0,0 +1,69 @@
+<?php
+include "classes.php";
+session_start();
+
+include "commonUI.php";
+$scores = array();
+startHTML("scores");
+
+
+$health = $_SESSION["playParty"]->_health;
+$members = $_SESSION["playParty"]->_livingMembers;
+
+if(strcmp($health, "Good") == 0){$scores[0] = 500 * $members;}
+else if(strcmp($health, "Fair") == 0){$scores[0] = 400 * $members;}
+else if(strcmp($health, "Poor") == 0){$scores[0] = 300 * $members;}
+else if(strcmp($health, "Bad") == 0){$scores[0] = 200 * $members;}
+
+$scores[1] =  50;
+$scores[2] = $_SESSION["playParty"]->_supplies->_oxen * 4;
+$scores[3] = ($_SESSION["playParty"]->_supplies->_wagonWheels +
+	$_SESSION["playParty"]->_supplies->_wagonAxle +
+	$_SESSION["playParty"]->_supplies->_wagonTongue) * 2;
+$scores[4] = $_SESSION["playParty"]->_supplies->_clothes * 2;
+$scores[5] = round($_SESSION["playParty"]->_supplies->_bait / 50);
+$scores[6] = round($_SESSION["playParty"]->_supplies->_food / 25);
+$scores[7] = round($_SESSION["playParty"]->_supplies->_money / 5);
+
+?>
+<p>
+Points for Survinving Members <?php echo $scores[0] ?><br>
```

Comment:
Make sure you have semicolons after each of your echoes. Every time you open another PHP tag it continues the context from the last one.